HTML5SDK/wwtlib/RenderContext.cs

```diff
@@ -1292,7 +1301,7 @@ public void FreezeView()
 
             TargetCamera = ViewCamera.Copy();
         }
-
+    
```

Comment:
It's not the most important thing in the world, but I always review my diffs before committing to Git to make sure I haven't accidentally introduced any spurious whitespace changes like this. It's easy to do in the WWT code since the whitespace is so inconsistent ...